tests/test_mssqlcliclient.py

```diff
@@ -121,8 +121,19 @@ def test_schema_table_views_and_columns_query():
         viewtest = "test_%s" % random_str()
         schematest = "test_%s" % random_str()
 
+        def drop_entities(mssqlcli_client):
+            list(mssqlcli_client.execute_query('DROP TABLE %s;' % tabletest1))
+            list(mssqlcli_client.execute_query('DROP TABLE %s;' % tabletest2))
+            list(mssqlcli_client.execute_query('DROP VIEW %s IF EXISTS;' % viewtest))
+            list(mssqlcli_client.execute_query('DROP TABLE %s;' % "."
+                                               .join([schematest, tabletest1])))
+            list(mssqlcli_client.execute_query('DROP SCHEMA %s;' % schematest))
+
         try:
             client = create_mssql_cli_client()
+
+            drop_entities(client)   # drop entities in beginning (in case tables exist)
```

Comment:
Would this cause any failures if the table doesn't exist?

Reply:
Yes--it should probably use a try/except block.

Reply:
Could we add an IF EXISTS wrapper to the utility queries instead?

Reply:
Otherwise we might fail dropping the first table while others persist, which causes the test to still fail overall due to preexisting objects or something.

Reply:
OH sorry I misunderstood your question. No, it won't fail if the table doesn't exist. But we should probably have something surrounding the client parameter in case that's set to None/not running properly.

Reply:
On second thought, I think current functionality is ok. We don't want that function running if the client doesn't get set.